.yarnrc.yml

```diff
@@ -1,2 +1,7 @@
-yarnPath: ".yarn/releases/yarn-berry.cjs"
 nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
```

Comment:
Zodat we `yarn interactive-upgrade` kunnen doen

Reply:
fijn